packages/db-model/src/db.ts

```diff
@@ -67,63 +65,23 @@ export function queryBuilder(connection: Knex, trx?: Knex.Transaction): Knex.Tra
 }
 
 /**
- * Push 'updated_at' key to params if the column exists in the table being updated.
- *
- * @param {Knex} connection
- * @param {string} table
- * @param {any} [params={}]
- * @returns {Promise<any>}
- */
-async function withTimestamp(connection: Knex, table: string, params: any = {}): Promise<any> {
-  const exists = await connection.schema.hasColumn(table, 'updated_at');
-
-  if (!exists || (exists && params.updatedAt)) {
-    return object.toSnakeCase(params);
-  }
-
-  return { ...object.toSnakeCase(params), updated_at: connection.fn.now() };
-}
-
-/**
- * Finds a record based on the params.
+ * Finds a single record based on the params.
  * Returns null if no results were found.
  *
  * @param {Knex} connection
  * @param {string} table
  * @param {object} [params={}]
  * @param {Knex.Transaction} [trx]
- * @returns {(Promise<T | null>)}
+ * @returns {Knex.QueryBuilder}
  */
-export async function get<T>(
+export function findFirst<T>(
   connection: Knex,
   table: string,
   params: object = {},
+  orderBy: string,
```

Comment:
There could be multiple.
```suggestion
  orderBy: string[],
```